### PR TITLE
Feature: Reload capes

### PR DESCRIPTION
**Pull request by Paxii**
Add a method for reloading capes on runtime.

Capes of users whose capes have already been downloaded will be refreshed once they get in range.

### CODE REVIEW

**Comment by MatthewSH** [comment]
@Marco-MC is there any way you could also review this PR to make sure nothing breaks? I'll take a look through some of the code, but you can probably test this faster than me.

File: com/capesapi/UserAPI.java [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static boolean hasCape(UUID uuid, String capeId) throws IOException {
 
   public static void addCape(UUID uuid, String capeId) throws IOException {
     String data = "{\"capeId\": \"" + capeId + "\"}";
-    URL url = new URL(baseURL + uuid.toString() + "/addCape");
+    URL url = new URL(baseURL + uuid.toString() + "/setCape");
```
Comment (MatthewSH) [inline com/capesapi/UserAPI.java]: This is wrong. The URL is still `addCape`. [Here's the API route](https://github.com/halfpetal/CapesAPI/blob/master/routes/api.php#L29)

Comment (MatthewSH) [inline com/capesapi/UserAPI.java]: This is wrong, please update this. 
The route is `addCape` according the the [API route here](https://github.com/halfpetal/CapesAPI/blob/master/routes/api.php#L29)

**Comment by Paxii** [comment]
@MatthewSH  I didn't rename the addCape method in UserAPI. I added a new one in CapeAPI and renamed it myself one commit later, no need to deprecate.

**Comment by MatthewSH** [comment]
@Paxii yes, I noticed that. That was my bad. I'll wait to see if @Marco-MC has anything to add and then we should be good to merge.

**Comment by Hexeption** [comment]
Just Checked it out and it looks good to me 👍

**Pull request merged**